e2e/shannon_relay_test.go

```diff
@@ -37,27 +39,28 @@ func Test_ShannonRelay(t *testing.T) {
 			// single endpoint, maintained by Grove.
 			name:      "should successfully relay eth_blockNumber for anvil",
 			reqMethod: http.MethodPost,
+			serviceID: "anvil",
+			reqPath:   "/v1/abcdef12",
```

Comment:
Ditto here: `reqPath` is the same for all the test cases. If at some point that changes, we can update the tests accordingly.

Reply:
> Ditto here: `reqPath` is the same for all the test cases. If at some point that changes, we can update the tests accordingly.

I take your point but curious what the harm is in having them as test variables vs singlularly defined vars? My preference is generally to have any variables used in the test defined in the test cases for both consistency and the possibility of using other values down the line.

Reply:
request path is not really `variable` here: all the tests use the same value. Declaring it as such makes it a lot easier to read the E2E tests by anyone who is not familiar with how PATH treats different parts of the request URL.

Reply:
> request path is not really `variable` here: all the tests use the same value. Declaring it as such makes it a lot easier to read the E2E tests by anyone who is not familiar with how PATH treats different parts of the request URL.

Got it; I do agree with this re: the request path.